README.md

```diff
@@ -530,6 +530,42 @@ Here's an example `.babelrc` file:
 }
 ```
 
+### Defining Static Modules
+
+This is a performance specific configuration which gives us two major performance benefits:
+
+#### 1. Reduce webpack's dev re-build time by 2x (or more)
+
+Webpack's rebuild time usually depend on the amount of code it needs to handle. Basically, most of those code is from NPM modules but not from your app's source code. With static modules, we are moving those NPM modules out of the webpack builder and serves directly.
```

Comment:
`usually depends`

Instead of:
> Basically, most of those code is from NPM modules but not from your app's source code. With static modules, we are moving those NPM modules out of the webpack builder and serves directly.

Maybe this:
> Most of the code it needs to handle is not your app's source code, but NPM modules. With static modules we are serving those NPM modules directly, instead of passing them through webpack.